src/Workspaces/VisualBasic/Portable/CodeStyle/VisualBasicCodeStyleOptionsProvider.vb

```diff
@@ -18,7 +18,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.CodeStyle
 
         Public ReadOnly Property Options As ImmutableArray(Of IOption) Implements IOptionProvider.Options
             Get
-                Return VisualBasicCodeStyleOptions.AllOptions
+                Return VisualBasicCodeStyleOptions.AllOptions.As(Of IOption)
```

Comment:
❔ Do we have tests to verify that all these conversions are successful?

❔ Do you know if this method converts the underlying array?

Reply:
Yes - without this change the global option service will not get these options from MEF imported option providers, and hence not consider these as serializable.